src/managers/config.py

```diff
@@ -141,6 +155,47 @@ def set_config_properties(self) -> None:
             file=self.config_file,
         )
 
+    @property
+    def tuning_parameters_valid(self) -> bool:
```

Comment:
nit, I don't think this qualifies as a property, better as a method 
```suggestion
    def is_tuning_parameters_valid(self) -> bool:
```

Reply:
Would be possible to argument for both sides here I'd say, no strong opinion though from my side -> changed it.